hub/powertoys/image-resizer.md

```diff
@@ -63,9 +63,3 @@ You can specify a directory in the filename format to group resized images into
 [Characters that are illegal in file names](/windows/win32/fileio/naming-a-file#file-and-directory-names) will be replaced by an underscore `_`.
 
 You can choose to retain the original _last modified_ date on the resized image or reset it at time of the resizing action.
-
-## See also
-
-- [Microsoft PowerToys overview](index.md)
```

Comment:
I added these links because we got customer feedback on this page asking how they install the resizer.

Reply:
Oh, okay. Seemed a bit random to me, but if there's a reason then I'll revert this.